python/doc/theory/numerical_methods/rosenblatt_transformation.rst

```diff
@@ -3,51 +3,54 @@
+the following context: the input random vector is :math:`\vect{X}` with marginal cumulative
+density functions  :math:`F_i` and copula :math:`C`. Nothing special is assumed about the
+copula.
+
+Let :math:`\vect{d}` be a  deterministic vector, let :math:`g(\vect{X}\,,\,\vect{d})` be the
+limit state function of the model and let :math:`\cD_f = \{\vect{X} \in \Rset^n \,/ \,
+g(\vect{X}\,,\,\vect{d}) \le 0\}` be an event whose probability :math:`P_f` is defined as:
+
+.. math::
     :label: PfX2
 
       P_f = \Prob{g(\vect{X}\,,\,\vect{d})\leq 0}=   \int_{\cD_f}  \pdf\, d\vect{x}
 
-  is to use the Rosenblatt transformation :math:`T` which is a
-  diffeomorphism from :math:`\supp{\vect{X}}` into the standard space
-  :math:`\Rset^n`, where distributions are normal, with zero mean, unit
-  variance and unit correlation matrix (which is equivalent in that
-  normal case to independent components).
+One way to evaluate the probability :math:`P_f` is to use the Rosenblatt transformation :math:`T` which is a
+diffeomorphism from the support of the distribution :math:`f_{\vect{X}}` into the standard
+space :math:`\Rset^n`, where distributions are normal, with zero mean, unit
+variance and unit correlation matrix (which is equivalent in that
+normal case to independent components).
 
-| Let us recall some definitions.
-| The *cumulative distribution function* :math:`F_{1,k}` of the
-  :math:`k`-dimensional random vector :math:`(X_1, \dots, X_k)` is
-  defined by its marginal distributions :math:`F_i` and the copula
-  :math:`C_{1,k}` through the relation:
+Let us recall some definitions.
+The *cumulative distribution function* :math:`F_{1,k}` of the
+:math:`k`-dimensional random vector :math:`(X_1, \dots, X_k)` is
+defined by its marginal distributions :math:`F_i` and the copula
+:math:`C_{1,k}` through the relation:
 
-  .. math:: F_{1,k}(x_1,\dots, x_k) = C_{1,k}(F_1(x_1),\dots, F_k(x_k))
+.. math::
+
+    F_{1,k}(x_1,\dots, x_k) = C_{1,k}(F_1(x_1),\dots, F_k(x_k))
 
-   with
+with
 
-  .. math::
+.. math::
     :label: subCopula
 
       C_{1,k}(u_1, \dots, u_k) = C(u_1, \dots, u_k, 1, \dots, 1)
 
 The *cumulative distribution function* of the conditional variable
 :math:`X_k|X_1, \dots, X_{k-1}` is defined by:
 
-.. math:: F_{k|1, \dots, k-1} (x_k|x_1, \dots, x_{k-1})   =  \displaystyle \frac{\partial^{k-1} F_{1,k}(x_1, \dots, x_k)}{\partial x_1 \dots \partial x_{k-1}} /\frac{\partial^{k-1} F_{1,k-1}(x_1, \dots, x_{k-1})} {\partial x_1 \dots \partial x_{k-1}}
+.. math::
+
+    F_{k|1, \dots, k-1} (x_k|x_1, \dots, x_{k-1}) = \frac{ \frac{\partial^{k-1} F_{1,k}(x_1, \dots, x_k)}{\partial x_1 \dots \partial x_{k-1}} }{ \frac{\partial^{k-1} F_{1,k-1}(x_1, \dots, x_{k-1})} {\partial x_1 \dots \partial x_{k-1}}}
 
-**Rosenblatt transformation**: Let :math:`\vect{X}` in :math:`\Rset^n`
+Rosenblatt transformation
+-------------------------
+
+Let :math:`\vect{X}` in :math:`\Rset^n`
```

Comment:
The equations below are not aligned correctly:

![image](https://user-images.githubusercontent.com/31351465/235193059-cb48f344-1dde-474e-b5f7-567019005b5c.png)

If you use:
```
.. math::

    T_1 : \Rset^n & \rightarrow \Rset^n\\
         \vect{X} & \mapsto     \vect{Y}=
         \left(
         \begin{array}{l}
           F_1(X_1)\\
           \dots \\
           F_{k|1, \dots, k-1}(X_k|X_1, \dots, X_{k-1})\\
           \dots \\
           F_{n|1, \dots, n-1}(X_n|X_1, \dots, X_{n-1})
         \end{array}
         \right) \\
    T_2 : \Rset^n & \rightarrow \Rset^n\\
         \vect{Y} & \mapsto     \vect{U}=
         \left(
         \begin{array}{l}
           \Phi^{-1}(Y_1)\\
           \dots \\
           \Phi^{-1}(Y_n)
         \end{array}
         \right)

```
you get:

![image](https://user-images.githubusercontent.com/31351465/235193233-ebdd508d-0232-4596-b37f-2647d2fb5046.png)

Reply:
Thanks!